src/it/scala/org/zalando/spearheads/innkeeper/routes/DeleteRouteSpec.scala

```diff
@@ -64,7 +64,7 @@ class DeleteRouteSpec extends FunSpec with BeforeAndAfter with Matchers {
             insertRoute("R2", ownedByTeam = "team1")
 
             val response = deleteSlashRoute(2, token)
-            response.status.shouldBe(StatusCodes.OK)
+            response.status should be(StatusCodes.OK)
```

Comment:
syntax sugar overload... I actually prefer the previous version. But it is a matter of personal preference